### PR TITLE
Constant and Popped Args for x86

### PR DESCRIPTION
**Pull request by CarolEidt**
Add support for constant arguments,
and handle the case where the callee must pop the stack.

### CODE REVIEW

**Comment by CarolEidt** [comment]
@dotnet/jit-contrib PTAL

File: src/jit/codegenxarch.cpp [hunk 1]
```diff
@@ -5915,6 +5918,15 @@ void CodeGen::genCallInstruction(GenTreePtr node)
             break;
         }
     }
+
+    // Is the caller supposed to pop the arguments?
+    if ((call->gtFlags & GTF_CALL_POP_ARGS) != 0)
```
Comment (pgavlin) [inline src/jit/codegenxarch.cpp]: Nit: could be simply `if ((call->gtFlags & GTF_CALL_POP_ARGS) != 0 && stackArgBytes != 0)`

**Comment by sivarv** [comment]
Looks good.

**Comment by pgavlin** [comment]
LGTM aside from one minor nit.

**Pull request merged**